optuna/visualization/matplotlib/_edf.py

```diff
@@ -23,7 +23,8 @@
 def plot_edf(study: Union[Study, Sequence[Study]]) -> "Axes":
     """Plot the objective value EDF (empirical distribution function) of a study with Matplotlib.
 
-    .. seealso::  optuna.visualization.plot_edf
+    .. seealso::
+        Please refer to :func:`~optuna.visualization.plot_edf` for an example.
```

Comment:
How about showing the module path in addition to the function name?
This sentence can be understood as if it leads users to the same reference.

```suggestion
        Please refer to :func:`optuna.visualization.plot_edf` for an example.
```

![image](https://user-images.githubusercontent.com/3255979/95045625-60897800-071d-11eb-8742-e5e20c71dc5c.png)

Reply:
In addition, how about adding the explain of the relationship between `optuna.visualization.matplotlib.plot_edf` and `optuna.visualization.plot_edf`? I think this message implies `optuna.visualization.plot_edf` be replaced with this method in the example, but it is not clearly stated.